client-spark/common/src/main/java/org/apache/spark/shuffle/RssSparkConfig.java

```diff
@@ -191,6 +191,10 @@ public class RssSparkConfig {
       createIntegerBuilder(new ConfigBuilder("spark.rss.client.unregister.thread.pool.size"))
           .createWithDefault(10);
 
+  public static final ConfigEntry<Integer> RSS_CLIENT_UNREGISTER_TIMEOUT_SEC =
+      createIntegerBuilder(new ConfigBuilder("spark.rss.client.unregister.timeout.sec"))
+          .createWithDefault(60);
```

Comment:
We can add a description using `withDescription`.

Reply:
This is a `ConfigEntry` from Spark, the `withDescription` is provided by Uniffle's own `ConfigOption`. I leave this consolidation to #1302.